pyk4a/calibration.py

```diff
@@ -18,69 +18,107 @@ class CalibrationType(IntEnum):
 
 class Calibration:
     def __init__(
-        self, device: PyK4A, config: Config, source_calibration: CalibrationType, target_calibration: CalibrationType,
+        self, handle: object, depth_mode: DepthMode, color_resolution: ColorResolution, thread_safe: bool = True
     ):
-        if isinstance(device, PyK4A):
-            self.device = device
-        else:
-            raise K4AException("Calibration instance created without a device of the proper class")
-        self.source_calibration = source_calibration
-        self.target_calibration = target_calibration
-        self.config = config
-
-    def convert_3d_to_3d(
```

Comment:
this method will be private, but  `depth_to_color_3d()` and  `color_to_depth_3d()` methods available instead